backend/app/controllers/tags_controller.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class TagsController < ApplicationController
+  # POST /tags
+  def create
+    tag = Tag.new(tag_params)
+    if tag.save
+      head :no_content
+    else
+      render json: { errors: tag.errors.full_messages }, status: :unprocessable_content
+    end
+  end
+
+  # PUT /tags/:id
+  def update
+    tag = Tag.find(params[:id])
+
+    if tag.update(update_tag_params)
+      head :no_content
+    else
+      render json: { errors: tag.errors.full_messages }, status: :unprocessable_content
```

Comment:
[お願い]

messageにしていただきたいです！
（errorsだと他のstatus codeのrenderと共通化しづらかったりするため）
```suggestion
      render json: { message: tag.errors.full_messages }, status: :unprocessable_content
```